flocker/provision/_install.py

```diff
@@ -104,6 +102,20 @@ def get_repository_url(distribution, flocker_version):
         raise UnsupportedDistribution()
 
 
+def get_repo_options(flocker_version):
+    """
+    Get a list of options for enabling necessary yum repositories.
+
+    :param bytes flocker_version: The version of Flocker to get options for.
+    :return: List of bytes for enabling (or not) a testing repository.
+    """
+    is_dev = not is_release(flocker_version)
```

Comment:
When I build the docs locally from your branch using `tox -e sphinx` this always gets passed version `1.0.0`...even when `python setup.py --version` returns none tag versions 

eg

```
(centos-latest-FLOC-1620 ? =)[~/.../HybridLogic/flocker]$ python setup.py --version
1.0.0pre1-109-gf99b2c4

```

```
(centos-latest-FLOC-1620 ? ▴62)[~/.../HybridLogic/flocker]$ python setup.py --version
1.0.0-82-gbe43106

```

So in the locally built docs this always returns an empty list.

How can I simulate both a dev and a release version locally?